music_album.rb

```diff
@@ -14,10 +14,10 @@ def can_be_archived?
 
   def to_hash
     {
-      genre: @genre,
+      genre: @genre.map(&:to_hash),
       author: @author,
       source: @source,
-      label: @label,
+      label: @label.map(&:to_h),
```

Comment:
In  `music_album.rb`, the error will  be sorted by changing the method name to `to_hash`. 
![image](https://user-images.githubusercontent.com/61404403/222568717-ad064135-b386-4659-8369-329f9f70e68c.png)